src/store/actions/index.js

```diff
@@ -0,0 +1,27 @@
+export const changeLogin = (check) => {
+  return {
+    type: "login_check",
+    payload: check,
+  };
+};
+
+export const changeModal = (active) => {
+  return {
+    type: "modal_active",
```

Comment:
타입은 대문자 스네이크 케이스로 써주셔야 합니다
```suggestion
    type: "MODAL_ACTIVE",
```

Reply:
넵 반영하였습니다